### PR TITLE
PIM-5348: fix role count on behats

### PR DESCRIPTION
**Pull request by juliensnz**
| Q                 | A
| ----------------- | ---
| Specs             | NA
| Behats            | NA
| Blue CI           | NA
| Changelog updated | Y
| Review and 2 GTM  | Y

### CODE REVIEW

File: features/Context/FixturesContext.php [hunk 1]
```diff
@@ -1529,10 +1529,13 @@ public function checkUserAssociationsCount($username, $count, $associationType)
         $this->refresh($user);
         $actualCount = null;
         if ($associationType == 'group') {
-            $actualCount = count($user->getGroupNames());
+            //We remove 1 for the "All" group which is not displayed to the user
+            $actualCount = count($user->getGroupNames()) - 1;
+            var_dump($user->getGroupNames());
```
Comment (jjanvier) [inline features/Context/FixturesContext.php]: nope :troll:

Comment (willy-ahva) [inline features/Context/FixturesContext.php]: Nope

**Comment by jjanvier** [comment]
giff diff > nope :p

gtm otherwise

**Comment by willy-ahva** [comment]
GTM after unnopized :p

**Pull request merged**